streams/src/test/java/org/apache/kafka/test/MockClientSupplier.java

```diff
@@ -56,8 +57,14 @@ public void setClusterForAdminClient(final Cluster cluster) {
         this.cluster = cluster;
     }
 
+    @SuppressWarnings("deprecation")
     @Override
```

Comment:
We should not suppress but `@deprecate` the method itself

Reply:
done.